components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Stack/Encoders/Models/EncodeableDictionary.cs

```diff
@@ -20,9 +19,9 @@ public class EncodeableDictionary : List<KeyDataValuePair>, IEncodeable {
         private const string kKeysIdentifier = "_Keys";
 
         /// <summary>
-        /// Identifier for the number of keys in the reversible format.
+        /// Identifier for the number of key-value pairs in the reversible format.
         /// </summary>
-        private const string kKeysCountIdentifier = "_KeysCount";
+        private const string kCountIdentifier = "_Count";
```

Comment:
Why the rename from kKeyCountIdentifier to kCountIdentifier?

Reply:
To generalize it more. It can be used by a decoder to know how many key-value pairs there are. It's the same number as for keys. But seemed a bit more general than having for instance a KeysCount and ValuesCount, when that number is always identical. Does that make sense @peterbryntesson ?

Reply:
Yes,that makes sense. Current code is still counting keys though.

Reply:
Yes, internally we only use it for that. I made it from an "Output" perspective. So it is more general in the output. I should have clarified that.